CHANGES.md

```diff
@@ -43,6 +43,7 @@ Features
 * [#608](https://github.com/java-native-access/jna/pull/608): Mavenize the build process - change parent and native pom artifactId/name to differentiate in IDE and build tools. [@bhamail](https://github.com/bhamail)
 * [#613](https://github.com/java-native-access/jna/pull/613): Make Win32Exception extend LastErrorException [@lgoldstein](https://github.com/lgoldstein).
 * [#613](https://github.com/java-native-access/jna/pull/614): Added standard 'Kernel32Util#closeHandle' method that throws a Win32Exception if failed to close the handle [@lgoldstein](https://github.com/lgoldstein).
+* [#621](https://github.com/java-native-access/jna/pull/621): COM: Added TYPEFLAGS constants for TYPEATTR in ITypeInfo.GetTypeAttr() [@SevenOf9Sleeper](https://github.com/SevenOf9Sleeper).
```

Comment:
Can you please try to make this look more like the other CHANGELOG entries please (and fix anything above that doesn't look the same)?

A correct one is:

```
* [#569](https://github.com/java-native-access/jna/pull/569): Added `com.sun.jna.platform.win32.Winspool.PRINTER_INFO_2` support. Added GetPrinter and ClosePrinter functions in `com.sun.jna.platform.win32.Winspool` - [@IvanRF](https://github.com/IvanRF).
```

There's an issue number, a dash before the Github name, all code is back-quoted and we mention namespaces, particularly important since JNA is not just Win32.